.agents/plugins/cve-resolution/skills/cve-resolution/SKILL.md

```
@@ -0,0 +1,247 @@
+```
+groups = {
+  "pillow:rhoai-2.25": [RHAIENG-4594, RHAIENG-4595, ...],
+  "tornado:rhoai-3.3": [RHAIENG-4278, RHAIENG-4279, ...],
+}
+```
+
+Sort groups by earliest due date, process first group.
+
+### Processing Steps
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)